egs/wsj/s5/steps/cleanup/create_segments_from_ctm_edits.py

```diff
@@ -0,0 +1,724 @@
+#!/bin/env python
+
```

Comment:
This script is *extremely* complicated.  I'm wondering if this was really the most straightforward way to do this.  Perhaps a two-pass strategy, with each individual pass simpler, would be better.